cjdns-snode/src/server.rs

```diff
@@ -560,6 +560,7 @@ mod nodes {
         pub(super) mut_state: RwLock<NodeMut>,
     }
 
+    #[derive(Clone)]
```

Comment:
What is this for? Looks very suspicious, because node state not supposed to be cloned, it should be shared instead, and all access is provided through an `Arc` reference only.

Reply:
well, it was a try to resolve a problem here: https://github.com/CJDNS-Development-Team/CJDNS/blob/4e3c1d0c5948852f9e8b8c96acd0f79fa9253b76/cjdns-snode/src/server/webserver.rs#L427

It's quite hard to create a `Wrapper(NodeMut)` with `&NodeMut`, without a clone. The only way to implement same logic without deriving `Clone` is to clone `NodeMut` field by field creating a new hardly cloned `NodeMut` and wrapping it. WDYT?

Reply:
a new implementation for the same logic: https://github.com/CJDNS-Development-Team/CJDNS/blob/11f1ab3639ab2e2cde310a298f5bc1a86fa92d70/cjdns-snode/src/server/webserver.rs#L381

Implementing `Clone` by hand without the trait.